src/main/java/nextstep/ladder/domain/Ladder.java

```diff
@@ -0,0 +1,21 @@
+package nextstep.ladder.domain;
+
+import java.util.ArrayList;
+import java.util.Objects;
+
+public class Ladder {
+
+    private ArrayList<Line> ladder = new ArrayList<>();
```

Comment:
```suggestion
    private List<Line> ladder = new ArrayList<>();
```
[리스코프치환원칙](https://ko.wikipedia.org/wiki/%EB%A6%AC%EC%8A%A4%EC%BD%94%ED%94%84_%EC%B9%98%ED%99%98_%EC%9B%90%EC%B9%99) 을 만족할 수 있도록 개선해보는건 어떤가요?